### PR TITLE
bff: revert project change

### PR DESCRIPTION
**Pull request by Chocimier**
bff by Oleg Mazonka and bff by Alex Pankratov are different piece of software, but one was replaced with other by [https://github.com/void-linux/void-packages/commit/2cb6148a216be2d4b4092dfde44f20edadb25a2d]().
Let's revert change and add the second as new package.

### CODE REVIEW

**Comment by maxice8** [comment]
please fix travis errors

File: srcpkgs/bff-apankrat/template [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+maintainer="Piotr Wójcik <chocimier@tlen.pl>"
+license="BSD-2-Clause"
+homepage="https://github.com/apankrat/bff"
+distfiles="https://github.com/apankrat/bff/archive/v${version}.tar.gz"
+checksum=8fff205c1c60f15c43b5613a73f1b6b7cac9f58c7c803eed4ed4c890a9374110
+
+do_install() {
+	vbin bff
+}
+
```
Comment (maxice8) [inline srcpkgs/bff-apankrat/template]: extra line

**Comment by Chocimier** [comment]
Tarball do not contain license, only address http://www.opensource.org/licenses/bsd-license.php . I put license into filesdir.

File: srcpkgs/bff-apankrat/files/LICENSE [hunk 2]
```diff
@@ -0,0 +1,9 @@
+Copyright <YEAR> <COPYRIGHT HOLDER>
```
Comment (maxice8) [inline srcpkgs/bff-apankrat/files/LICENSE]: The license needs to be from the source code

**Comment by Chocimier** [comment]
changed

**Pull request merged**